frontend/packages/fusion-frontend/package.json

```diff
@@ -46,7 +46,7 @@
   },
   "dependencies": {
     "@vaadin/common-frontend": "^0.0.12",
-    "lit": "^2.0.0-rc.2"
+    "lit": "^2.0.0"
```

Comment:
Here `lit` should also be moved to `peerDependencies`:
```
"dependencies": {
  "@vaadin/common-frontend": "^0.0.12"
},
"peerDependencies": {
  "lit": "^2.0.0"
}
```

Reply:
If we release a new `@vaadin/common-frontend` first we might as well update the version of that here (though it's not necessarily needed since we use the caret here).

Reply:
done